pkg/controller/plan/kubevirt.go

```diff
@@ -1760,7 +1759,27 @@ func (r *KubeVirt) podVolumeMounts(vmVolumes []cnv.Volume, configMap *core.Confi
 			EmptyDir: &core.EmptyDirVolumeSource{},
 		},
 	})
-
+	if vm.LUKS.Name != "" {
+		err = r.isSecretExist(vm.LUKS.Name, r.Plan.Spec.TargetNamespace)
```

Comment:
this assumes the secret with the keys exists on the target namespace but how is it supposed to get there? there should be a step in the migration that ensures the secret exists on the target namespace..

Reply:
it should be created by the user/UI. we don't know the passphrases

Reply:
it's a problem to expect the client to interact with the destination provider - we shouldn't assume things are set on the target namespace by other entities

the way I see it is as follow:
1. the user specifies keys
2. Forklift creates a secret along with the plan (on the same namespace)
3. during the migration, Forklift creates a secret on the target namespace (this can happen right after Forklift creates the target namespace)

and then when we get here, we'll find the secret on the target namespace